File: other/hello.py
import cgi, time, os

def main():
	form = cgi.FieldStorage()
	print("<p>Hello, world!</p>")
	print("<p>The time is: " + time.ctime() + "</p>")
	print("<html><body>")
	if not "name" in form:
		print("<h1>Hello!</h1>")
		print("<p>You did not give me your name!</p>")
	else:
		print("<h1>Hello, %s</h1>" % form["name"].value)
	print("</body></html>")

main()

File: other/hello.php
<?php
	$name = $_POST['name'];
	// header('Content-Type: text/plain');
	if ($name == "") {
		echo "<h1>Hello, world!</h1>";
		echo "<h2>You didn't enter your name.</h2>";
	} else {
		echo "<h1>Hello $name</h1>";
	}
?>
